modules/checking-deployment-yaml-files.adoc

```diff
@@ -8,13 +8,22 @@
 .Procedure
 
 * Run the following command to check the build-time and deploy-time violations of your security policies in YAML deployment files:
-//TODO: Add link to security policies section
 +
-[source,terminal,subs="+quotes"]
+[source,terminal]
 ----
-$ roxctl deployment check --file=_<yaml_filename>_
+$ roxctl deployment check --file=<yaml_filename> \// <1>
+--namespace=<cluster_namespace> \// <2>
+--cluster=<cluster_name_or_id> \// <3>
+--verbose <4>
 ----
 +
+--
+<1> For the `<yaml_filename>`, specify the YAML file with one or more deployments to send to Central for policy evaluation. You can specify multiple YAML files to send to Central for policy evaluation.
+<2> For the `<cluster_namespace>`, specify a namespace to enhance deployments with context information such as network policies, role-based access controls (RBACs) and services for deployments that do not have a namespace in their specification. The namespace defined in the specification is not changed. The default value is `default`.
+<3> For the `<cluster_name_or_id>`, specify the cluster name or ID that you want to use as the context for the evaluation to enable extended deployments with cluster-specific information.
+<4> By enabling the `--verbose` flag, you receive additional information for each deployment during the policy check. The extended information includes the RBAC permission level and a comprehensive list of network policies that is applied.
```

Comment:
The additional output will also show up in the JSON report (`--json`). It will be always there, no matter whether `--verbose` is given or not.